src/leiningen/ezbake.clj

```diff
@@ -12,6 +12,7 @@ as defined below.
 Actions:
   stage      Generate and stage ezbake artifacts.
   build      Build native packages from staged artifacts.
+  json       Generate ezbake manifest json file.
```

Comment:
I agree with Haus' comment about choosing JSON or YAML, but not both.

And then, is there a better name for this action?  JSON is just a text format, so it doesn't really tell me anything about what data it will contain.

I might take inspiration from leiningen here and call this `deps` unless there's a better name.